projects/Generic/linux/linux.x86_64.conf

```diff
@@ -2607,7 +2625,7 @@ CONFIG_RMI4_F30=y
 CONFIG_SERIO=y
 CONFIG_ARCH_MIGHT_HAVE_PC_SERIO=y
 CONFIG_SERIO_I8042=y
-CONFIG_SERIO_SERPORT=y
+CONFIG_SERIO_SERPORT=m
```

Comment:
why did you change that from `y` to `m`?

Reply:
My mistake. From my build tree. Just reran from a clean upstream, and fixed. Reconfirmed the remaining is the default correct .config